core/src/Platforms/android/TokenCacheAccessor.cs

```diff
@@ -30,6 +30,7 @@
 using Microsoft.Identity.Client;
 using Microsoft.Identity.Core.Cache;
 using System.Collections.ObjectModel;
+using Microsoft.Identity.Core.Helpers;
 
 namespace Microsoft.Identity.Core
 {
```

Comment:
Please add comemnts, especially if this is multi-threaded or not (I assume not?)

Reply:
please keep scope  of pr to changes in pr

Reply:
pls. take in the code style discussion. IMO if the base interface has the right amount of description we might be good. But seems relevant with a few well crafted words

---
In reply to: [199550027](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199550027) [](ancestors = 199550027)

Reply:
pls. take in the code style discussion

---
In reply to: [199979992](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r199979992) [](ancestors = 199979992)